src/utils/problemUtils.ts

```diff
@@ -0,0 +1,8 @@
+import kebabCase = require("lodash.kebabcase");
```

Comment:
Use `import` instead of `require`.

Reply:
I use it in this way because of the issue of lodash. T_T

Reply:
OK, it's fine.